tests/BuildCommandTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Umpirsky\ListGenerator;
+
+use Umpirsky\ListGenerator\Builder\Builder;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+use PHPUnit\Framework\TestCase;
+
+class BuildCommandTest extends TestCase
+{
+    /**
+     * @var commandTester
+     */
+    private $commandTester;
+
+    /**
+     * @var vfsStreamDirectory
+     */
+    private $vfsDirectory;
+
+    protected function setUp()
+    {
+        $builder = new Builder(new ImporterInstance(), 'importer_path');
+        $command = $builder->get('build');
+
+        $this->commandTester = new CommandTester($command);
+    }
+
+    protected function tearDown()
+    {
+        $format = ['json', 'csv', 'html', 'txt', 'xliff', 'xml', 'yaml', 'php'];
+        foreach($format as $formatName) {
+            $importerFile = glob(__DIR__.'/../importer_path/en/*.'.$formatName);
+            foreach($importerFile as $file) {
+                @unlink($file);
+            }
+        }
+
+        @rmdir(__DIR__.'/../importer_path/en');
+        @rmdir(__DIR__.'/../importer_path');
+    }
+
+    public function testBuildCommandWithFormatJson()
```

Comment:
All this test cases can be one with `@dataProvider` which passes all possible formats.